browser/cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
Is this file boilerplate that we can remove?

Reply:
This can be removed if a certain flag it set, but I think it'd be better to keep this to have the option open to modify behavior of Cypress in the future if needed.